manifests/r/Rustlang/Rustup/1.24.3/Rustlang.Rustup.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Rustlang.Rustup
+PackageVersion: 1.24.3
+PackageLocale: en-US
+Publisher: Rustlang
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Rustlang
+PackageName: Rustup
```

```suggestion
PackageName: "Rustup: the Rust toolchain installer"
```
This must match Control Panel.